code/modules/mob/living/carbon/human/species/grey.dm

```diff
@@ -40,14 +38,8 @@
 	. = ..()
 
 	if(method == REAGENT_TOUCH)
-		if(H.wear_mask)
-			to_chat(H, "<span class='danger'>Your [H.wear_mask] protects you from the acid!</span>")
-			return
-
-		if(H.head)
-			to_chat(H, "<span class='danger'>Your [H.wear_mask] protects you from the acid!</span>")
+		if(H.head?.flags & THICKMATERIAL && H.wear_suit?.flags & THICKMATERIAL) // fully pierce proof clothing is also water proof!
```

Comment:
```suggestion
		if((H.head?.flags & THICKMATERIAL) && (H.wear_suit?.flags & THICKMATERIAL)) // fully pierce proof clothing is also water proof!
```
Not required to make it work. But it makes it a bit more clear that this is a bitwise operation and not a logical operation